src/main.ts

```diff
@@ -31,7 +38,7 @@ function buildNpmDependencies(): any {
 }
 
 const command: Command = {
-	description: 'test your application',
+	description: 'this command will implicitly build your application and then run tests against that build',
```

Comment:
As per the discussion about capitalisation, full stops and short descriptions, do you think it's worth changing this to `Build and test the application.`?